packages/asset-buyer/src/asset_buyer.ts

```diff
@@ -76,8 +76,8 @@ export class AssetBuyer {
     ): AssetBuyer {
         const provider = providerUtils.standardizeOrThrow(supportedProvider);
         assert.isWebUri('sraApiUrl', sraApiUrl);
-        const chainId = options.chainId || constants.DEFAULT_ASSET_BUYER_OPTS.chainId;
-        const orderProvider = new StandardRelayerAPIOrderProvider(sraApiUrl, chainId);
+        // HACK: asset-buy will be deleted, but do not pass in chainId to allow everything to compile.
```

Comment:
@fragosti will @dave4506 be able to find all these hacks easily? E.g., is there a keyword or list of things he should grep for?

Reply:
This is the only hack in this PR, and we are deleting the entire package (task is tracked in Asana so we don't forget etc...). So I think we're good.